provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -262,11 +399,10 @@ tasks:
         - ROSA_CLUSTER_NAME_1
         - ROSA_CLUSTER_NAME_2
         - AURORA_CLUSTER
-        - AURORA_REGION
     cmds:
       - task: common:datasetprovider
       - task: create-aurora
-      - task: fetch-aurora-endpoint
+      - task: fetch-{{ ternary "global" "single" (eq .CROSS_DC_MODE "ASYNC") }}-aurora-endpoint
```

Comment:
how to switch between `global` and `single` aurora tasks in this case ? do we have an additional task variable ?

Reply:
looks like we are controlling that with `CROSS_DC_MODE=ASYNC` variable ? Its kind of neat but it was not evident to the user in the first look. Can we leave some breadcrumbs to indicate that's what is happening ?

Reply:
We can add a comment there

Reply:
added a comment to indicate that in the `default:` task